test/integration_test.py

```diff
@@ -292,7 +292,9 @@ def _test_manifest(self, catalog: CatalogName):
             ('compact', self._check_manifest, 1),
             ('full', self._check_manifest, 3),
             ('terra.bdbag', self._check_terra_bdbag, 1),
-            ('terra.pfb', self._check_terra_pfb, 1),
+            # FIXME: PFB test is disabled because of a failed prod build
+            #        https://github.com/DataBiosphere/azul/pull/3149
```

Comment:
Like I said, reference ticket, not PR. This isn't rocket science, may try to nail it next time?